test/e2e/suite_test.go

```diff
@@ -77,6 +80,16 @@ var (
 	pushModeClusterLabels = map[string]string{"sync-mode": "Push"}
 )
 
+func init() {
+	var pollIntervalValue, pollTimeoutValue int
+
+	flag.IntVar(&pollIntervalValue, "poll-interval-value", 300, "Interval value of seconds for poll operation")
+	flag.IntVar(&pollTimeoutValue, "poll-timeout-value", 5, "Value of seconds which the poll operation times out")
```

Comment:
The default value of those two flags may be mixed up.

Reply:
I am sorry I has not finished testing and the pull request was not ready at first. 
Now I finish testing including the value passed from command to program .
Besides I try to use `/kind cleanup`to replace the label, But the origin label `kind bug` has not removed. Can you help to modify it ?

Reply:
Updated.